cirq/google/__init__.py

```diff
@@ -45,7 +45,9 @@
 )
 
 from cirq.google.gate_sets import (
-    XMON,)
+    XMON,
+    SYCAMORE_GATESET,
```

Comment:
Since Sycamore is the chip, and the gate is SYC, maybe we should call this SYC_GATESET?